contracts/DistributionCreator.sol

```diff
@@ -320,9 +346,31 @@ contract DistributionCreator is UUPSHelper, ReentrancyGuardUpgradeable {
 
     /// @notice Returns the list of all the distributions that were or that are going to be live at
     /// a specific epoch
-    function getDistributionsForEpoch(uint32 epoch) external view returns (ExtensiveDistributionParameters[] memory) {
+    function getDistributionsForEpoch(
+        uint32 epoch
+    ) external view returns (ExtensiveDistributionParameters[] memory searchDistributions) {
         uint32 roundedEpoch = _getRoundedEpoch(epoch);
-        return _getPoolDistributionsBetweenEpochs(address(0), roundedEpoch, roundedEpoch + EPOCH_DURATION);
+        (searchDistributions, ) = _getPoolDistributionsBetweenEpochs(
+            address(0),
+            roundedEpoch,
+            roundedEpoch + EPOCH_DURATION,
+            0,
+            type(uint32).max
+        );
+    }
+
+    /// @notice Returns the list of all the distributions that were or that are going to be live at
+    /// a specific epoch
+    /// @param skip Disregard distibutions with a global index lower than `skip`
+    /// @param first Limit the lenght of the returned array to `first`
+    /// @dev Similar to `getDistributionsForEpoch(uint256 epoch)` with additional parameters to prevent out of gas error
```

Comment:
Maybe we can just keep in notice this comment we have in dev and remove the notice comment:

```
/// @notice Similar to `getDistributionsForEpoch(uint256 epoch)` with additional parameters to prevent out of gas error
/// @dev skip Disregard distibutions with a global index in the `distributionList` array lower than `skip`
/// @dev first Limit the length of the returned array to `first`
```